guides/trailblazer/2.0/01-operation-basics.md

```diff
@@ -10,14 +10,14 @@ redirect_from: "/guides/trailblazer/2.0/01-getting-started-with-operation.html"
   {{ "app/blog_post/operation/create.rb:failure:../trailblazer-guides:operation-01" | tsnippet : "impl" }}
 
   ~~~6
-  The *Operation* is the central concept of the Trailblazer architecture. It is a simple service object that embraces and orchestrates all the business logic necessary to accomplish a certain task, such as creating a blog post, or updating a user.
+  The *Operation* is the central concept of the Trailblazer architecture. It's a simple service object that encapsulates and executes all the business logic necessary to accomplish a certain task, such as creating a blog post, or updating a user.
```

Comment:
If you define the logic in the step, yes, but that's not the idea of TRB operations. Most logic as finding the models, contract/validation, persisting, post-processing, etc. happens in different objects that the operation has no knowledge about, except for their APIs, of course. Hence my "orchestration" wording. Makes sense?